### PR TITLE
eos_cli_config_gen: adding gnmi template and fixing TA config for CVaaS

### LINKED ISSUE #302: Add management api gnmi template
**Issue Type**
- Template enhancement

**Is your feature request related to a problem? Please describe.**
Openconfig and OCTA template aka `management api gnmi ` config is missing currently. Would be nice to add to leverage openconfig/EOS-native telemetry

**Describe the solution you'd like**
the j2 template should be something like:
```
{% if management_api_gnmi is defined and management_api_gnmi is not none %}
management api gnmi
  transport grpc avd
{%     if management_api_gnmi.enable_vrfs is defined -%}
{%         for vrf in management_api_gnmi.enable_vrfs | arista.avd.natural_sort if vrf is ne 'default' %}
   !
      vrf {{ vrf }}
         no shutdown
{%             endif %}
{%-         endfor %}
{%     if management_api_gnmi.octa is defined -%}
      provider eos-native
{%             endif %}
```

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### LINKED ISSUE #430: Port and token path is incorrect for CVaaS config in TerminAttr template
## Issue Type

- Bug Report

## Summary

When streaming EOS states to CVaaS the dst port is incorrectly set to 9910 (which is the port used on on-prem CVP)  instead of 443 and the token path is set to the default one which is changed after initial onboarding

## Role or Module Name

eos_cli_config_gen

### Fix

The correct config would look something like this:

```
daemon TerminAttr
   exec /usr/bin/TerminAttr -cvaddr=apiserver.arista.io:443 -cvcompression=gzip -taillogs -cvauth=token-secure,/tmp/cv-onboarding-token -smashexcludes=ale,flexCounter,hardware,kni,pulse,strata -ingestexclude=/Sysdb/cell/1/agent,/Sysdb/cell/2/agent -cvvrf=MGMT
```

### PR DESCRIPTION
**Pull request by noredistribution**
## Change Summary
Added management api gnmi template and fixed TerminAttr CVaaS config

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [x] Documentation content changes
- [ ] Other (please describe):

## Related Issue(s)
Fixes #430 
Fixes #302 

## How to test
I've used the `dc-fabric-deploy-cvp.yml` playbook from https://github.com/arista-netdevops-community/ansible-webinar-february-2020 

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://www.avd.sh/docs/contributing/) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed ([`pre-commit`](https://www.avd.sh/docs/installation/development/#python-virtual-environment), `make linting` and `make sanity-lint`).
- [ ] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly

### CODE REVIEW

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/terminattr-cloud.cfg [hunk 1]
```diff
@@ -1,7 +1,7 @@
 !RANCID-CONTENT-TYPE: arista
 !
 daemon TerminAttr
-   exec /usr/bin/TerminAttr -cvaddr=apiserver.corp.arista.io:9910 -cvcompression=gzip -taillogs -cvauth=token-secure,/tmp/onboardingtoken1 -smashexcludes=ale,flexCounter,hardware,kni,pulse,strata -ingestexclude=/Sysdb/cell/1/agent,/Sysdb/cell/2/agent  -cvvrf=mgt
+   exec /usr/bin/TerminAttr -cvaddr=apiserver.corp.arista.io:443 -cvcompression=gzip -taillogs -cvauth=token-secure,/tmp/cv-onboarding-token -smashexcludes=ale,flexCounter,hardware,kni,pulse,strata -ingestexclude=/Sysdb/cell/1/agent,/Sysdb/cell/2/agent  -cvvrf=mgt
```
Comment (carlbuchmann) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/terminattr-cloud.cfg:4]: @noredistribution please validate generated output!

**Review by carlbuchmann — COMMENTED** [review]
LGTM - please validate CI output, and we can merge!

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/terminattr-cloud.cfg (hunk 1 above)
Comment (noredistribution) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/terminattr-cloud.cfg:4]: yes, that's the correct config, works on my CVaaS tenant

**Review by titom73 — CHANGES_REQUESTED** [review]
Can you please build GNMI's documentation file as well ? In a similar way of PR #443

**Review by titom73 — APPROVED** [review]

**Pull request merged**